plugins/package-managers/gradle-model/src/main/kotlin/Extensions.kt

```diff
@@ -22,14 +22,14 @@ package org.ossreviewtoolkit.plugins.packagemanagers.gradlemodel
 import OrtDependency
 
 /**
- * The type of this Gradle dependency. In case of a project, it is "Gradle". Otherwise it is "Maven" unless there is no
- * POM, then it is "Unknown".
+ * The type of this Gradle dependency. In case of a project, it is [projectType]. Otherwise it is "Maven" unless there
+ * is no POM, then it is "Unknown".
  */
-val OrtDependency.dependencyType: String
-    get() = if (isProjectDependency) {
-        "Gradle"
-    } else {
-        pomFile?.let { "Maven" } ?: "Unknown"
+fun OrtDependency.getIdentifierType(projectType: String) =
```

Comment:
While at it, should we turn this into `OrtDependency.getIdentifier(projectType: String): Identifier`, so the caller code is less redundant?

Reply:
The `gradle-model` project that contains this code does not (and should not) depend on ORT's `model` project, so that refactoring is unfortunately not possible.

Reply:
Correct, but I regard that to be less of a problem as we hopefully can drop the "legacy" Gradle implementation in the foreseeable future.